variables.tf

```diff
@@ -2,6 +2,11 @@
 # Module Level Variables
 ##############################################################################
 
+variable "name" {
```

Comment:
what is the reasoning behind the variable name change? `vpc_name` -> `name` ? Just thinking can we avoid the breaking change here?

Reply:
We can but will be out of sync with what we have and will have to carry that forward with the other things that use it.  Not sure if there is a standard surrounding the `name` aspect but having `vpc` within is a little intuitive.  

Definitely understand the breaking aspect.

Reply:
As there is probably nobody consuming this module, I am OK with making this breaking change (and bumping the major version to 2.0.0) to get the module in sync with what is being used internally.